modules/users/client/components/utils/i18n-angular-load.js

```diff
@@ -0,0 +1,28 @@
+import '@/config/lib/i18n';
```

Comment:
This can be deduplicated (rebase and use the file which is already in master).

Already discussed with @agatatalita .